lib/daos/driverDao.js

```diff
@@ -70,21 +78,31 @@ export const acceptBooking = async bookingDetail =>
  * update dao method to update driver details.
  * @author rushabh
  * @date 2022-03-07
- * @param {any} ride_status
- * @param {any} ride_statrted_at
- * @param {any} ride_ended_at
+ * @param {Enumerator('pending','accepted','completed','cancelled')} rideStatus
+ * @param {Date} rideStarted_at
+ * @param {Date} rideEndedAt
  * @returns {any}
  */
 export const updateBookingAttempts = async bookingDetail => {
-    const { booking_attempt_id, ...restDetail } = bookingDetail;
+    const { bookingAttemptId, ...restDetail } = bookingDetail;
     return booking_attempts.update(
         {
             ...restDetail
         },
         {
             where: {
-                id: booking_attempt_id
+                id: bookingAttemptId
             }
         }
     );
 };
+
+/**
+ * update dao method to update driver details.
+ * @author rushabh
+ * @date 2022-03-07
+ * @param {number} bookingAttemptId
+ * @returns {any}
+ */
+export const findBookingAttempt = async bookingAttemptId =>
+    booking_attempts.findOne({ where: { id: bookingAttemptId } });
```

Comment:
If this is the pk can we use https://sequelize.org/master/manual/model-querying-finders.html `findByPk`? Would be nice to try no?

Reply:
Also shouldn't we convert this to rawResponse?

Reply:
Done.